### PR TITLE
Feature/send updated dependencies stats

### PR DESCRIPTION
**Pull request by adelkahomolova**
## Description

Send statistics of updated dependencies to the dashboard.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have added new practice to practice list in README.md.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by adelkahomolova** [comment]
> I'd suggest not using `*Dto` interfaces in any other files except `DashboardReporter` directly because we can change the structure as we will need there.

So should I duplicate the interface so it has just different naming? Or? @prokopsimek

File: src/practices/Java/JavaDependenciesVersionMajorLevel.ts [hunk 1]
```diff
@@ -55,7 +56,7 @@ export class JavaDependenciesVersionMajorLevel extends PracticeBase {
     return latestVersionsJson;
   }
 
-  setData(pkgsToUpdate: PkgToUpdate[]): void {
+  setData(pkgsToUpdate: UpdatedDependencyDto[]): void {
     this.data.details = [{ type: ReportDetailType.table, headers: ['Name', 'New', 'Current'], data: pkgsToUpdate }];
```
Comment (prokopsimek) [inline src/practices/Java/JavaDependenciesVersionMajorLevel.ts]: Missing major statistics data?

Comment (adelkahomolova) [reply src/practices/Java/JavaDependenciesVersionMajorLevel.ts]: It sends data both from Major Level and Minor and from Patch Level as the method setData() is called both in JavaDependenciesVersionMajorLevel, and in JavaDependenciesVersionMinorPatchLevel

File: src/reporters/DashboardReporter.ts [hunk 2]
```diff
@@ -49,15 +49,24 @@ export class DashboardReporter implements IReporter {
       dxScore: { value: dxScore.value, points: dxScore.points },
     };
     const securityVulnerabilitiesPractice = practicesAndComponents.find((p) => p.practice.id === 'JavaScript.SecurityVulnerabilities');
+
+    const updatedDependenciesMinorPatch = practicesAndComponents.find(
+      (p) => p.practice.id === 'JavaScript.DependenciesVersionMinorPatchLevel',
+    );
+    const updatedDependenciesMajor = practicesAndComponents.find((p) => p.practice.id === 'JavaScript.DependenciesVersionMajorLevel');
+
```
Comment (prokopsimek) [inline src/reporters/DashboardReporter.ts:57]: move to `forof`

Comment (prokopsimek) [reply src/reporters/DashboardReporter.ts:57]: read as 
```
const updatedDepsInPractices = practicesAndComponents.filter((p) => p.practice.data?.statistics?.updatedDependencies);
const allUpdatedDeps = _.flatten(updatedDepsInPractices)
```
+ security issues

**Pull request merged**